components/dashboard/src/projects/NewProject.tsx

```diff
@@ -210,21 +193,6 @@ export default function NewProject() {
         });
     }
 
-    const grantReadOrgPermissions = async () => {
-        try {
-            await openAuthorizeWindow({
-                host: "github.com",
-                scopes: ["read:org"],
-                onSuccess: () => {
-                    updateReposInAccounts();
-                    updateOrgsState();
-                }
-            })
-        } catch (error) {
-            console.log(error);
-        }
-    }
```

Comment:
**Praise**: Super cool that we no longer need this secondary/orthogonal authorization flow! 🙌 (I personally found it super confusing to have both "Reconfigure" and "Grant org scope" -- must have been even worse for users who don't actually know what these two things even mean 🙈)